get-message-service-bus-queue-peek-lock-renew-sequential-convoy/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
-    "title": "Azure Service Bus: Get message from a queue with token renewal - Peek lock renew pattern",
+    "title": "Azure Service Bus: Sequential convoy pattern",
 
-    "description": "This workflow template triggers when a message is available in a Service Bus queue. The workflow gets the message from the queue, locks the message for processing, and renews the lock token until processing completes. The message is deleted from the queue after processing completes. This pattern helps you make sure that the workflow processes the message only once and that the message doesn't get lost if a failure happens during processing.",
+    "description": "This workflow template triggers when a message is available in a Service Bus session enabled queue. The workflow processes related messages from a specific session on the same workflow instance, creating different workflow instances for each session in the queue.",

```

Comment:
I'm having trouble parsing some of this. I'm not sure which specific item creates difference workflow instances for each session in the queue?